conf.py

```diff
@@ -59,7 +59,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'AT-TPC Analysis'
+project = 'AT-TPC_Analysis'
```

Comment:
The underscore is necessary to compile in pdf format. Atleast for me, the space causes sphinx to look for latex documents titled only AT-TPC which do not exist. This may be a linux thing caused by the space but the underscore isn't ideal for ReadTheDocs. Does it compile on your computer (mac)?